Composer/plugins/localPublish/.eslintrc.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
-    extends: ['../../.eslintrc.js'],
-    parserOptions: {
-        project: './tsconfig.json',
-        tsconfigRootDir: __dirname,
-    },
-    rules: {
-        'security/detect-non-literal-fs-filename': 'off',
-    },
+  extends: ['../../.eslintrc.js'],
+  parserOptions: {
+    project: './tsconfig.json',
+    tsconfigRootDir: __dirname,
+  },
+  rules: {
+    // 'security/detect-non-literal-fs-filename': 'off',
```

Comment:
why did this get removed?

Reply:
because Chris commented on this before, and seems like it's not necessary, so I remove this which default is 'on'.